front/app/api/background_jobs/useTrackBackgroundJobs.ts

```diff
@@ -25,7 +31,30 @@ const useTrackBackgroundJobs = ({
     polledJobs?.data &&
     polledJobs.data.length > 0 &&
     polledJobs.data.every((job) => job.attributes.failed);
-  return { active, failed };
+  const errors: CLError[] =
+    polledJobs?.data && polledJobs.data.length > 0
+      ? polledJobs.data.flatMap((job) => {
+          if (!!job.attributes.last_error_message) {
+            const error_string = job.attributes.last_error_message;
+            // When a bulk_import_ error - there will be additional params after a #
+            const splitErrorParams = error_string.split('#');
+            if (splitErrorParams.length > 1) {
+              const params = JSON.parse(splitErrorParams[1]) as JobErrorParams;
```

Comment:
Maybe these `split` and `parse` could also be done on the BE. The fact that we send to the FE JSON (HTTP response) with another JSON inside (these error params) seems a bit too complicated.

But it's definitely not a requirement. Feel free to merge!